src/components/HeroCard/herocard.css

```diff
@@ -0,0 +1,78 @@
+.hero {
+  position: relative;
+  display: flex;
+  height: 100vh;
+  width: 100%;
+  padding: 16px 16px;
+}
+
+.hero > img {
+  position: absolute;
+  width: 80%;
+  height: 50%;
+  top: 62px;
+  left: 31px;
+  z-index: -10;
+  animation: run 0.5s alternate;
+}
+
+.hero__content {
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  position: relative;
+  padding: 0 12px;
+  text-align: center;
+  z-index: -5;
+}
+
+.hero__content h2 {
+  font-size: 2rem;
+  font-weight: 300;
+  line-height: 28px;
+  color: #000000;
+  animation: run 1s alternate;
+}
+
+.hero__content h1 {
+  font-size: 2.5rem;
+  color: #2d6180;
+  font-weight: 700;
+  line-height: 56px;
+  letter-spacing: 0.03em;
+  animation: run 1.5s alternate;
+}
+
+.hero__content h3 {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #000000;
+  animation: run 1.5s alternate;
+}
+
+@media screen and (min-width: 768px) {
+  .hero > img {
+    position: absolute;
+    width: 40%;
+    height: 80%;
+  }
+
+  .hero__content {
+    position: absolute;
+    width: 50vw;
+    top: 30vh;
+    right: 20px;
+  }
+}
```

Comment:
thay lại thành

@media screen and (min-width: 768px) {
  .hero {
    display: flex;
    align-items: center;
    text-align: left;
    height: calc(100vh - 48px); /* 48px = header height */
    padding: 16px;
  }

  .hero > img {
    max-width: 50%; /* responsive */
    max-height: calc(100vh - 48px);
  }

  .hero__content {
    flex-grow: 1; /* occupy the remaining width */
  }
}